### PR TITLE
72 composable for webviewlink to events

### LINKED ISSUE #72: Composable for webview/link to events
Create a composable that has a web-view and a link to the upcoming events on emojipedia and add it to its section on the homepage underneath a upcoming events label

### PR DESCRIPTION
**Pull request by Steffenriv**
Composable for events is done, also finished eventListScreen to view all events and topics. Added tests, and navigation between screens and also navigation to event url. TODO: replace static data with mock data, will be done in another issue.
Closes #72

### CODE REVIEW

File: app/src/androidTest/kotlin/com/platform/platform/openemoji/navigation/NavigationTest.kt [hunk 1]
```diff
@@ -68,4 +68,23 @@ class NavigationTest {
...
+        }
+        // Make sure we're in the home screen.
+        composeTestRule.onNodeWithTag("bottomNavigationBarHome")
+            .performClick()
+        // Test that clicking on show more, will navigate to the event list screen.
+        composeTestRule.onNodeWithTag("showMoreEvents")
+            .performClick()
+        composeTestRule.onNodeWithTag("eventListScreen").assertExists()
+        // Test back button navigation, sleep first in order to load it.
+        Thread.sleep(1000)
```
Comment (OleAndreasER) [inline app/src/androidTest/kotlin/com/platform/platform/openemoji/navigation/NavigationTest.kt]: Don't sleep in tests. You should instead wait for a node to appear.

File: app/src/main/kotlin/com/platform/openemoji/events/EventCard.kt [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+@Composable
+fun EventCard(event: Event) {
+    val screenWidth = LocalConfiguration.current.screenWidthDp.dp
+    val context = LocalContext.current
+    val eventIntent = remember { Intent(Intent.ACTION_VIEW, Uri.parse(event.url)) }
+
+    val painter =
+        rememberImagePainter(
+            data = event.img,
```
Comment (OleAndreasER) [inline app/src/main/kotlin/com/platform/openemoji/events/EventCard.kt]: rememberImagePainter is deprecated. I don't know if this is an issue at all, so you can decide if it is.

**Review by OleAndreasER — CHANGES_REQUESTED** [review]
This is good stuff.

File: app/src/main/kotlin/com/platform/openemoji/events/EventCard.kt (hunk 2 above)
Comment (Steffenriv) [reply app/src/main/kotlin/com/platform/openemoji/events/EventCard.kt]: replaced with AsyncImage

File: app/src/androidTest/kotlin/com/platform/platform/openemoji/navigation/NavigationTest.kt [hunk 3]
```diff
@@ -82,9 +82,10 @@ class NavigationTest {
...
-        Thread.sleep(1000)
-        composeTestRule.onNodeWithTag("eventListBackButton", useUnmergedTree = true)
-            .performClick()
+        composeTestRule.onNodeWithTag(
+            "eventListBackButton",
+            useUnmergedTree = true,
+        ).performClick()
         composeTestRule.onNodeWithTag("homeScreen").assertExists()
     }
 }
```
Comment (OleAndreasER) [inline app/src/androidTest/kotlin/com/platform/platform/openemoji/navigation/NavigationTest.kt]: Sorry for the late update. You forgot to remove the comment about sleep. Also, was the sleep not needed then?

Comment (Steffenriv) [reply app/src/androidTest/kotlin/com/platform/platform/openemoji/navigation/NavigationTest.kt]: It's fine it is easter after all. Sleep was not needed indeed, will remove the comment now!

**Pull request merged**